go/vt/vttablet/onlineddl/executor.go

```diff
@@ -2858,6 +2949,7 @@ func (e *Executor) reviewRunningMigrations(ctx context.Context) (countRunnning i
 
 		uuidsFoundRunning[uuid] = true
 
+		_ = e.updateMigrationUserThrotleRatio(ctx, uuid, currentUserThrottleRatio)
```

Comment:
nit: I think throttle is typod in the method name above

Reply:
fixed